### PR TITLE
sign up interface done

### PR DESCRIPTION
**Pull request by AyoFayomi**

### CODE REVIEW

**Comment by rammyblog** [comment]
@AyoFayomi  This is the requirement for signup
  firstName: Joi.string().min(6).required(),
   lastName: Joi.string().min(6).required(),
    email: Joi.string().min(6).required().email(),
    password: Joi.string().min(6).required()

Kindly adjust to meet the above conditions

File: frontend/pages/Signup.js [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+} from "@chakra-ui/react";
+
+export default function App() {
+  return (
+    <Box bg="gray.100" align="center" p={30} w="100vw">
+      <Flex justify="space-around" h="93.5vh" align="center">
+        <Box display={{ sm: "none", lg: "flex" }}>
+          <Image
+            src={registerimage}
+            // src="https://payfoodo.herokuapp.com/static/media/register.fed4c349.svg"
```
Comment (rammyblog) [inline frontend/pages/Signup.js]: Remove this comment

File: frontend/pages/Signup.js [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+            <Formik
+              initialValues={{
+                fullname: "",
+                username: "",
+                email: "",
+                password: "",
+                Address: "",
+              }}
+              onSubmit={(values) => {
+                // alert(JSON.stringify(values, null, 2));
```
Comment (rammyblog) [inline frontend/pages/Signup.js]: You can reload to the homepage for now

Comment (AyoFayomi) [reply frontend/pages/Signup.js]: i really don't know how to do that

Comment (AyoFayomi) [reply frontend/pages/Signup.js]: done already

**Comment by rammyblog** [comment]
Can you change the filename from signup.js to register.js

**Review by rammyblog — APPROVED** [review]

**Pull request merged**